src/main/java/org/synyx/urlaubsverwaltung/person/basedata/PersonBasedata.java

```diff
@@ -0,0 +1,94 @@
+package org.synyx.urlaubsverwaltung.person.basedata;
+
+public class PersonBasedata {
+
+    private final int personId;
+    private final String personnelNumber;
+    private final String additionalInformation;
+
+    private final String niceName;
+    private final String gravatarURL;
+    private final String email;
```

Comment:
Should we move these fields in another PR?

Reply:
why do we have these here and in the `Person` class?

Reply:
I think this is only for the presentation at `person/person_basedata`

Reply:
We would like to stay clean on domain level, controller needs to compose dto with person data